tests/integration_test.rs

```diff
@@ -268,3 +274,82 @@ async fn test_pypi_ignore(
     let pack_result = pixi_pack::pack(pack_options).await;
     assert_eq!(pack_result.is_err(), should_fail);
 }
+#[rstest]
+#[tokio::test]
+async fn test_run_packed_executable(options: Options, required_fs_objects: Vec<&'static str>) {
+    let temp_dir = tempfile::tempdir().expect("Couldn't create a temp dir for tests");
```

Comment:
```suggestion
    let temp_dir = tempfile::tempdir().unwrap();
```

unwrap is fine imo as this is not user facing